test2.js

```diff
@@ -0,0 +1,19 @@
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( 'hey' )
+}
+
+
+var foo88 = '' ;
+
+if(foo88  === "b88") {
```

Comment:
Codacy has a fix for the issue: [Multiple spaces found before '==='.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26923)
```suggestion
if(foo88 === "b88") {
```